app/jobs/cache_analytics_job.rb

```diff
@@ -370,30 +370,30 @@ def self.monitoree_count(analytic_id, active_monitoring, category_type, category
   end
 
   # Monitoree flow over time and monitoree action summary
-  def self.all_monitoree_snapshots(analytic_id, monitorees, jurisdiction_id)
+  def self.all_monitoree_snapshots(analytic_id, monitorees, subjur_ids)
     counts = []
     MONITOREE_SNAPSHOT_TIME_FRAMES.map do |time_frame|
       WORKFLOWS.map do |workflow|
         counts.append(MonitoreeSnapshot.new(
                         analytic_id: analytic_id,
                         time_frame: time_frame,
                         new_enrollments: monitorees.where(isolation: workflow == 'Isolation')
-                        .enrolled_in_time_frame(time_frame)
-                        .size,
-                        transferred_in: Transfer.where_assoc_exists(:patient,
-                                                                    isolation: workflow == 'Isolation')
-                          .with_incoming_jurisdiction_id(jurisdiction_id)
-                          .in_time_frame(time_frame)
-                          .size,
+                                                   .enrolled_in_time_frame(time_frame)
+                                                   .size,
+                        transferred_in: Transfer.where(to_jurisdiction_id: subjur_ids)
```

Comment:
Can you add a comment here explaining this logic for other devs?

Reply:
sorry, thought I replied to this earlier already but is this language good? https://github.com/SaraAlert/SaraAlert/pull/666/commits/c8ab8b7020581ca15a6a952ae454223981abf36b

Reply:
Yep! Thanks.